src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/StorageAnalyticsLogEntry.cs

```diff
@@ -139,23 +137,19 @@ public BlobPath ToBlobPath()
             else
             {
                 // assuming key is "/account/container/blob"
-                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;                
+                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;
                 path = startPos != -1 ? RequestedObjectKey.Substring(startPos + 1) : String.Empty;
             }
 
             if (String.IsNullOrEmpty(path))
             {
-                throw new FormatException("Failed to parse RequestedObjectKey property of the log entry. " +
-                    "It should be in one of the supported formats: " +
-                    @"""https://account.blob.core.windows.net/container/blob"", or" +
-                    @"""/account/container/blob""");
+                return null;
```

Comment:
It looks like `BlobLogListener` handles null already (I mentioned this in the issue as well) -- https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobLogListener.cs#L67-L75

Although I don't see any tests for that scenario -- I'll look into that now.

I was going to handle ThrottlingError directly, but then saw that there are 20+ other statuses (https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/storage-analytics-logged-operations-and-status-messages). If another customer finds that one of these also doesn't contain a valid path, we'll be right back in this code patching for that one. It seemed smarter to ignore any log entries that do not contain a valid `BlobPath` as we can't operate on them anyway.